pkgs/applications/version-management/git-and-tools/git-secrets/default.nix

```diff
@@ -1,40 +1,32 @@
-{ stdenv, lib, fetchFromGitHub, makeWrapper, git }:
+{ stdenv, fetchFromGitHub, makeWrapper, git, gawk, gnused }:
 
-let
+stdenv.mkDerivation rec {
+  name = "git-secrets-${version}";
   version = "1.2.1";
-  repo = "git-secrets";
-
-in stdenv.mkDerivation {
-  name = "${repo}-${version}";
 
   src = fetchFromGitHub {
-    inherit repo;
-    owner = "awslabs";
-    rev = "${version}";
+    owner  = "awslabs";
```

Comment:
Please don't align these in this manner.

Reply:
fixed